command/issues/list/utils.go

```diff
@@ -0,0 +1,29 @@
+package list
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/deepsourcelabs/cli/deepsource/issues"
+)
+
+func FilterIssuesByPath(path string, issuesData []issues.Issue) ([]issues.Issue, error) {
```

Comment:
```suggestion
func filterIssuesByPath(path string, issuesData []issues.Issue) ([]issues.Issue, error) {
```
Probably also add a comment about the purpose of this function.